### ISSUE TITLE
False positive "Not enough arguments for format string" with starred expression

### ISSUE THREAD

**Issue by jwilk** [PyCQA/pylint#957]
### Current behavior

pylint emits "Not enough arguments for format string" for this valid program:

``` python
meat = ['spam', 'ham']
print('%s%s%s' % ('eggs', *meat))
```
### pylint --version output

```
No config file found, using default configuration
pylint 1.5.6, 
astroid 1.4.6
Python 3.5.2rc1 (default, Jun 13 2016, 09:33:26) 
[GCC 5.4.0 20160609]
```

**Comment by PCManticore** [comment]
Thanks, looking into it!

**Comment by wolfy1339** [comment]
Seeing the same issue when referencing a function from self.
Example code:
```python
class foo(object):
    def la(self, arg1):
        [...]

    def ma(self):
        self.la(arg1)
```

**Comment by PCManticore** [comment]
@wolfy1339 I don't know what you mean, that snippet of code does not emit the error we are talking about in this issue.

**Comment by wolfy1339** [comment]
Oops, you're right
